schemas/computation.schema.tpl.json

```diff
@@ -0,0 +1,84 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/Computation",
+    "_extends": "/core/research/activity.schema.tpl.json",
```

Comment:
@apdavison : activity.schema.tpl.json as the superclass of Computation and ProtocolExecution?

Reply:
Would activity.schema.tpl.json hold only the following properties?
+ name
+ description
+ input
+ output

Reply:
I'm planning a PR against core to implement this. Probably also parameterSet, studyTarget; possibly semanticallyAnchoredTo, startedAtTime, endedAtTime, wasInformedBy